node/v2/call.js

```diff
@@ -139,6 +148,12 @@ function writeCallReqInto(body, buffer, offset) {
     // flags:1 -- filled in later after argsrw
     offset += bufrw.UInt8.width;
 
+    if (body.ttl === 0) {
```

Comment:
Is this coerced to a positive integer somewhere? Shouldn't we be checking if it's > 0? What if the user doesn't pass in a ttl at all? Then it's certainly not exactly 0, but it is an invalid ttl.